Cargo.lock

```diff
@@ -20,15 +20,15 @@ checksum = "c71b1793ee61086797f5c80b6efa2b8ffa6d5dd703f118545808a7f2e27f7046"
 
 [[package]]
 name = "accesskit"
-version = "0.17.0"
+version = "0.17.1"
```

Comment:
I believe according to https://github.com/linebender/rfcs/blob/main/rfcs/0005-version-matrix.md#cargotoml-matches-cargolock, Cargo.toml should be updated to match these patch versions.

Reply:
It should be fixed now by running `cargo upgrade`

Reply:
We should always specify the patch version in Cargo.toml, yes. So we never should have a dependency of the form `accesskit = "0.17"`.

That's largely a thing for consistency, but it does have advantages for e.g. when we publish, largely in case any user decides to use the MSRV-aware resolver or minimal-versions.